File: ejemplos/rps.py
import random
import sys

user_options = ["rock", "paper", "scissors"]

try:
    user_choice = sys.argv[1]
    if user_choice in user_options:
        computer_choice = random.choice(["rock", "paper", "scissors"])
        print(computer_choice)
        if user_choice == computer_choice:
            print("Even")
        elif user_choice == "rock" and computer_choice == "paper":
            print("Computer win")
        elif user_choice == "rock" and computer_choice == "scissors":                
            print("You win")
        elif user_choice == "paper" and computer_choice == "rock":
            print("You win")
        elif user_choice == "paper" and computer_choice == "scissors":                
            print("Computer win")
        elif user_choice == "scissors" and computer_choice == "rock":
            print("Computer win")
        elif user_choice == "scissors" and computer_choice == "paper":
            print("You win")
    else:
        print("Invalid value")
except IndexError:
    print("Invalid format")

File: ejemplos/rps.js
var yourChoice;
var computersChoice;

function rps(yourChoice, computersChoice) {

    var1 = "Tie";
    var2 = "You win";
    var3 = "Computer win";

    if (yourChoice === computersChoice){
        return var1;
    } else if (yourChoice == "rock" && computersChoice == "paper"){
        return var3;
    } else if (yourChoice == "rock" && computersChoice == "scissors"){
        return var2;
    } else if (yourChoice == "paper" && computersChoice == "scissors"){
        return var3;
    } else if (yourChoice == "paper" && computersChoice == "rock"){
        return var2;
    } else if (yourChoice == "scissors" && computersChoice == "rock"){
        return var3;
    } else if (yourChoice == "scissors" && computersChoice == "paper"){
        return var2;
    }
}

console.log(rps("scissors","paper"));